docs/ingestion/hadoop.md

```diff
@@ -23,6 +23,15 @@ sidebar_label: "Hadoop-based"
   ~ under the License.
   -->
 
+:::caution[Deprecated]
+
+Hadoop-based ingestion deprecated. Use [SQL-based ingestion](../multi-stage-query/index.md) instead of MapReduce or [MiddleManager-less ingestion using Kubernetes](../development/extensions-core/k8s-jobs.md) instead of YARN.
```

Comment:
```suggestion
Hadoop-based ingestion is deprecated. Use [SQL-based ingestion](../multi-stage-query/index.md) instead of MapReduce or [MiddleManager-less ingestion using Kubernetes](../development/extensions-core/k8s-jobs.md) instead of YARN.
```